java/src/main/java/com/powsybl/dataframe/network/NetworkDataFramesBusBreakerView.java

```diff
@@ -0,0 +1,512 @@
+package com.powsybl.dataframe.network;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.BooleanSeriesMapper;
+import com.powsybl.dataframe.DataframeElementType;
+import com.powsybl.dataframe.DoubleSeriesMapper;
+import com.powsybl.iidm.network.*;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.commons.lang3.tuple.Triple;
+
+import java.util.*;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.ToDoubleFunction;
+import java.util.function.ToIntFunction;
+import java.util.stream.Stream;
+
+import static com.powsybl.dataframe.MappingUtils.ifExistsDouble;
+import static com.powsybl.dataframe.MappingUtils.ifExistsInt;
+
+public final class NetworkDataFramesBusBreakerView {
```

Comment:
This class add a lot of code duplication, I would keep everything in  NetworkDataFrames with 2 BUS_BREAKER_VIEW_MAPPER in addition to MAPPERS passing a boolean to createMappers(boolean busBreakerView) and all elements related methods generators(boolean busBreakerView)

Reply:
I don't know how I could pass the boolean to createmapper() because it is created once, Am I wrong ?

Reply:
You mean that mappers will be recreated each time we call get_elements() ?

Reply:
No you can just have 2 caches map, one for busview and an other one for busbreakerview:
```java
    public static NetworkDataframeMapper getDataframeMapper(DataframeElementType type, boolean busBreakerView) {
        return busBreakerView ? BUS_BREAKER_VIEW_MAPPERS.get(type) : MAPPERS.get(type);
    }
```